ResearchKit/Localized/en.lproj/ResearchKit.strings

```diff
@@ -341,27 +341,31 @@
 
 /* Tremor test active task. */
 "TREMOR_TEST_TITLE" = "Tremor Activity";
-"TREMOR_TEST_INTENDED_USE" = "This activity measures the tremor of your hands in various positions. Perform this test with your more affected hand.";
-"TREMOR_TEST_INTRO_2_TEXT" = "Hold the phone as shown in the image below and find a place where you can sit comfortably for the duration of this task.";
-"TREMOR_TEST_INTRO_2_DETAIL_%" = "You will be asked to perform %@ while sitting with the phone in your hand.";
+"TREMOR_TEST_INTENDED_USE" = "This activity measures the tremor of your hands in various positions. Find a place where you can sit comfortably for the duration of this task.";
+"TREMOR_TEST_INTRO_2_DEFAULT_TEXT" = "Hold the phone in your more affected hand as shown in the image below.";
+"TREMOR_TEST_INTRO_2_RIGHT_HAND_TEXT" = "Hold the phone in your right hand as shown in the image below.";
```

Comment:
For the tapping test we went with all caps for RIGHT and LEFT to help highlight which hand to use.